src/main/scala/wdl4s/cwl/model/CWLType.scala

```diff
@@ -0,0 +1,55 @@
+package broad.cwl.model
+
+import enumeratum._
+import mouse.boolean._
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.string._
+import eu.timepit.refined._
+import eu.timepit.refined.auto._
+
+sealed abstract class CWLType(override val entryName: String) extends EnumEntry
```

Comment:
OK.  We should probably discuss this, I think some people in the group may have strong opinions.  😄